src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -101,6 +94,19 @@ void createStationWithDuplicateName() {
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        // given
+        createNewStation("강남역");
+        createNewStation("역삼역");
+
+        // when
+        ExtractableResponse<Response> getResponse = getSavedStations();
+
+        // then
+        assertAll(
+                ()->assertThat(getResponse.statusCode()).isEqualTo(HttpStatus.OK.value()),
+                ()->assertThat(getResponse.jsonPath().getList(".").size()).isEqualTo(2),
+                ()->assertThat(getResponse.jsonPath().getList("name", String.class)).containsExactly("강남역", "역삼역")
+        );
```

Comment:
```suggestion
        지하철_역_목록_존재한다(getResponse, 리스트("강남역", "역삼역"));
```

이렇게도 해볼수 있겠네요 :thinking: 
(저의 관점이므로 참고해주세요~)